### ISSUE TITLE
Identify health disparities in Charlottesville

### ISSUE THREAD

**Issue by robert-rotzin** [UVA-DSI/2017-doathon#43]
[//]: # "=======================
Instructions for posting issues:
(1) Review what is already there. Perhaps a comment to an existing issue would be more appropriate than opening a new one?  Issues can be a topic idea, a cry for help, a to-do, or a discussion / question about the event itself.
(2) Write your post (below this text blob) using Markdown (as per https://guides.github.com/features/mastering-markdown/ ) or just plain text.
(3) Don't worry about these introductory lines - you can leave or delete them, as they won't display anyway (you can check this via Preview).
(4) Hit the 'Submit new issue' button.
============================"

**Comment by jcruser** [comment]
Sounds interesting!

**Comment by eih2nn** [comment]
I'd love to work on this.

**Comment by rrobeva** [comment]
Yes, this is of interest to me (one of several). Cross referencing medical conditions would be interesting/useful.

**Comment by nbhaip** [comment]
Counties Dashboard
https://www.vdh.virginia.gov/omhhe/hoi/dashboards/counties

**Comment by jcruser** [comment]
Dental info:
-basic stats: http://www.vhcf.org/data/statistics-and-research-on-dental-access/ 
-health resources & services admin: https://datawarehouse.hrsa.gov/tools/analyzers/hpsafind.aspx  - no data for Charlottesville or Ablemarle county
-children's dental health in va (2011): http://www.pewtrusts.org/en/research-and-analysis/fact-sheets/2011/05/11/childrens-dental-health-virginia
-dental use/changes/coverage (1996-2004) -https://meps.ahrq.gov/data_files/publications/cb17/cb17.pdf
-veteran's dental encounters (not public) - https://www.data.va.gov/dataset/dental-encounter-system-des
-Virginia oral heath coalition: http://www.vaoralhealth.org/RESOURCES/DataandResearch.aspx

**Comment by eih2nn** [comment]
Idea for Social Determinants of Health research: Overlay maps of house pricing (a proxy for income/socioeconomic status) and CAT routes (color coded by frequency of buses at each stop in a specific loop) to determine how well current public transit serves low income communities.

**Comment by robert-rotzin** [comment]
https://virginiawellbeing.com/
https://www.policymap.com/
http://www.countyhealthrankings.org/

Have to sign up for a free account but with tons of data:
https://www.communitycommons.org/maps-data/

**Comment by alizaidia** [comment]
Website with a lot of interesting visualizations and a few data sources we could look more into: https://datausa.io/profile/geo/albemarle-county-va/#category_hospital_care_for_medicare_patients
